docs/concept/wal.md

```diff
@@ -0,0 +1,56 @@
+# Wal Format
+
+![](/assets/wal-format.jpg)
+
```

Comment:
There are many grammar mistakes here. Could you check them at https://app.grammarly.com/ ?